api/tests/unit/application/usecases/checkUserBelongsToScoOrganization_test.js

```diff
@@ -0,0 +1,78 @@
+const { expect, sinon, domainBuilder } = require('../../../test-helper');
+const useCase = require('../../../../lib/application/usecases/checkUserBelongsToScoOrganization');
+const membershipRepository = require('../../../../lib/infrastructure/repositories/membership-repository');
+
+describe('Unit | Application | Use Case | CheckUserBelongsToScoOrganization', () => {
+
+  beforeEach(() => {
+    membershipRepository.findByUserIdAndOrganizationId = sinon.stub();
+  });
+
+  context('When user is in a SCO organization', () => {
+
+    it('should return true', async () => {
+      // given
+      const userId = 1234;
+
+      const organization = domainBuilder.buildOrganization({ type: 'SCO', isManagingStudents: true });
+      const membership = domainBuilder.buildMembership({ organization });
+      membershipRepository.findByUserIdAndOrganizationId.resolves([membership]);
+
+      // when
+      const response = await useCase.execute(userId, organization.id);
+
+      // then
+      expect(response).to.equal(true);
+    });
+
+    it('should return true with several memberships', async () => {
```

Comment:
`should return true for user with several memberships` ?
J'ai eu l'impression qu'on allait retourner true et les memberships

Reply:
my bad j'ai pas trouvé mieux pour le nom du test :/

Reply:
modifié !